### ISSUE TITLE
doesn't compile anymore when servo option activated

### ISSUE THREAD

**Issue by Windaub57** [MagoKimbra/MarlinKimbra#167]
sketch\src\MK_Main.cpp: In function 'void servo_init()':

sketch\src\MK_Main.cpp:813:5: error: 'servo' was not declared in this scope

     servo[0].attach(SERVO0_PIN);

     ^

sketch\src\MK_Main.cpp:848:18: error: 'STOW_Z_SERVO' was not declared in this scope

     STOW_Z_SERVO();

                  ^

sketch\src\MK_Main.cpp: In function 'bool set_probe_deployed(bool)':

sketch\src\MK_Main.cpp:1837:7: error: 'servo' was not declared in this scope

       servo[Z_ENDSTOP_SERVO_NR].move(z_servo_angle[deploy ? 0 : 1]);

       ^

exit status 1

**Comment by MagoKimbra** [comment]
Board?
Num servos you define?

**Comment by Windaub57** [comment]
arduino mega 2560


#define NUM_SERVOS 1


________________________________
De : MagoKimbra <notifications@github.com>
Envoyé : mercredi 28 décembre 2016 22:30
À : MagoKimbra/MarlinKimbra
Cc : Windaub57; Author
Objet : Re: [MagoKimbra/MarlinKimbra] doesn't compile anymore when servo option activated (#167)


Board?
Num servos you define?

-

**Comment by MagoKimbra** [comment]
And Z_ENDSTOP_SERVO_NR ?

**Comment by Windaub57** [comment]
#define Z_ENDSTOP_SERVO_NR 0


________________________________
De : MagoKimbra <notifications@github.com>
Envoyé : mercredi 28 décembre 2016 23:45
À : MagoKimbra/MarlinKimbra
Cc : Windaub57; Author
Objet : Re: [MagoKimbra/MarlinKimbra] doesn't compile anymore when servo option activated (#167)


And Z_ENDSTOP_SERVO_NR ?

-

**Comment by Windaub57** [comment]
looks like I opened my ticket under the wrong firmware version. I'm using the latest one : Version 4.3.09_dev

**Comment by Windaub57** [comment]
sorry

**Comment by MagoKimbra** [comment]
Aaaa ok....

**Comment by MagoKimbra** [comment]
Do you have ramps 1.4?
I checked width ramps and all ok!!

**Comment by Windaub57** [comment]
yes I have ramps 1.4. I also tried the online configurator and got the exact same pb. same after upgrading to arduino1.8.


I'll try again.

**Comment by Windaub57** [comment]
New try

I took a brand fresh copy.

I only changed :
cartesian -> delta
#define Z_ENDSTOP_SERVO_NR 0
#define NUM_SERVOS 1
#define Z_PROBE_PIN         ORIG_Z_MIN_PIN

=> compil

Still get the same error. Any last ideas :) ?

**Comment by MagoKimbra** [comment]
and #define ENABLE_SERVOS?
